homeassistant/components/stream/__init__.py

```diff
@@ -27,6 +27,7 @@
 
 STREAM_SERVICE_SCHEMA = vol.Schema({
     vol.Required(CONF_STREAM_SOURCE): cv.string,
+    vol.Required(CONF_PROTOCOL, default='tcp'): cv.string,
```

Comment:
Optional if you provide a default...

Reply:
Also it's not `cv.string` , there are two options